cfme/cloud/provider.py

```diff
@@ -230,6 +258,8 @@ def _form_mapping(self, create=None, **kwargs):
             })
         return data_dict
 
+OpenStackProvider.add_type_map(OpenStackProvider)
```

Comment:
Yes, I looked at that - was one detail I couldn't get to work, so I just did it this way for "getting it working" phase